auth_user_rights_wizard/views/hr_employee_view.xml

```diff
@@ -24,35 +24,35 @@
         </record>
 
 
-        <record id="view_access_right_wizard" model="ir.ui.view">
-            <field name="name">hr.employee.user.right.form</field>
-            <field name="model">hr.employee.user.right</field>
-            <field name="arch" type="xml">
-                <form string="Access Rights">
-                    <group>
-                        <field name="employee_id" invisible="1"/>
-                        <field name="group_id" required="1" widget="many2many_tags" options="{'no_create': True, 'no_open': True}"/>
-                    </group>
-                    <footer>
-                        <button string="Assign" type="object" name="action_assign_rights" class="btn-primary" />
-                        <button string="Cancel" special="cancel" class="btn-secondary"/>
-                    </footer>
-                </form>
-            </field>
-        </record>
+<!--        <record id="view_access_right_wizard" model="ir.ui.view">-->
+<!--            <field name="name">hr.employee.user.right.form</field>-->
+<!--            <field name="model">hr.employee.user.right</field>-->
+<!--            <field name="arch" type="xml">-->
+<!--                <form string="Access Rights">-->
+<!--                    <group>-->
+<!--                        <field name="employee_id" invisible="1"/>-->
+<!--                        <field name="group_id" required="1" widget="many2many_tags" options="{'no_create': True, 'no_open': True}"/>-->
+<!--                    </group>-->
+<!--                    <footer>-->
+<!--                        <button string="Assign" type="object" name="action_assign_rights" class="btn-primary" />-->
+<!--                        <button string="Cancel" special="cancel" class="btn-secondary"/>-->
+<!--                    </footer>-->
+<!--                </form>-->
+<!--            </field>-->
+<!--        </record>-->
```

Comment:
Please state why you want to keep this code, with a code comment. Thanks.

Reply:
Nothing in particular, I don't know if we will need to implement it again, but I can remove it